app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<div class="user-post">
+   <div class="user-card">
+    
+    <div class="user-image">
+          <img src="<%= @user.photo_link %>" alt="<%= @user.name %> photo" class="img-fluid rounded-circle">
+      </div> 
+   <div class="user-info">
+  <h3><%= @user.name %></h3>
+  <p class="comment">Number of posts: <%= @user.posts.count %></p>
+  </div>
+  </div>
+
+  <div class="user-bio">
+  <p>Bio: <%= @user.bio %></p>
+  </div>
+  
+ <% @user.posts.each_with_index do |post, index| %>
+<div class="allposts-user">
+    <h3>Post <%= index + 1 %></h3>
```

Comment:
- [ ] The post on the user show page should be a link to the post show page, not just plain text